google/cloud/storage/emulator/emulator.py

```diff
@@ -561,7 +561,8 @@ def objects_copy(src_bucket_name, src_object_name, dst_bucket_name, dst_object_n
         flask.request, dst_metadata, dst_media, dst_bucket, True, None, dst_rest_only
     )
     db.insert_object(flask.request, dst_bucket_name, dst_object, None)
-    dst_object.patch(flask.request, None)
+    if len(flask.request.data):
```

Comment:
would `if flask.request.data:` be more pythonic?

Reply:
yes it would, thank you :)